src/components/input-fields/input-digits.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+    <div class="input-digit__component d-grid gap-1 w-100 mb-2">
+        <div class="input-digit d-grid w-auto gap-1 rounded-pill">
+            <input class="input-digit__field"
+                :id="id"
+                type="text"
+                v-model="inputValueObj"
+                @input="(input) => $emit('input', input.target.value)"
+            />
+            <div class="input-digit__arrows ms-auto">
+                <btn class="btn btn-icon input-digit__arrow-up" @click="incrementValue">
+                    <PhCaretUp :size="10" color="#fff" />
+                </btn>
+                <btn class="btn btn-icon input-digit__arrow-down" @click="decrementValue">
+                    <PhCaretDown :size="10" color="#fff" />
+                </btn>
```

Comment:
Add `text-white` class to btn's
Remove `color="#fff"` attributes from icons

Reply:
Worked on this.